src/demos/Inputs.vue

```diff
@@ -469,88 +466,29 @@ export default class Inputs extends Vue {
   passwordInputPlaceholder = "Placeholder";
 
   options = [
-    {
```

Comment:
these options were for our Image Input Picker so that demo will break if you delete these